src/version.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module utils/version
+ */
+
+/* global window, global */
+
+import log from './log';
+import { version } from 'ckeditor5/package.json';
+
+const windowOrGlobal = typeof window === 'object' ? window : global;
+
+if ( windowOrGlobal.CKEDITOR_VERSION ) {
+	/**
+	 * This error is thrown when the base {@link module:core/editor/editor~Editor `Editor`}
+	 * class has been loaded twice (or more), a result of either:
+	 *
+	 * * the build containing two (or more) different versions of the {@link module:core/editor/editor~Editor `Editor`}
```

Comment:
This sounds like this error occurs when you load `ClassicEditor and `InlineEditor` together. It'd be good to keep it oriented on duplicating packages. It may be easier to understand. Duplicated `Editor` class is just a result of that.

Reply:
We could also give recipies how to solve this:

1. Run `rm -rf node_modules && npm i` first.
2. Check whether you're using up to date versions of all CKEditor 5 packages.
3. `npm@5+` is known for installing packages in a non-optimal, non-deduped way. You may try downgrading npm to version 4.